### PR TITLE
Ana/add amount and time to vote

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

WIP because I would like to investigate a bit if it is easy to get the cast date for votes & deposits

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Was investigating if it is also possible to get votes amounts for motions, but it seems this code only gives the votes to become council members:

```
const votes = await api.derive.council.votes()
console.log(votes.map(vote => {
  return JSON.stringify({
    address: vote[0],
    vote: JSON.stringify(vote[1])
  })
}))
```

Going to look at treasuries

**Comment by Bitcoinera** [comment]
Ready to merge then

**Comment by faboweb** [comment]
> This is what seems to be promoted by the W3F itself since they think the blockchain should always store minimal data

the blockchain yes, but the node could automatically index like in Cosmos...

File: api/lib/schema.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ const typeDefs = gql`
     id: String
     voter: NetworkAccount
     option: String
+    amount: Coin # Substrate only
+    time: String # Substrate only
```
Comment (faboweb) [inline api/lib/schema.js:76]: we don't get time in the end, am I wrong?

Comment (faboweb) [reply api/lib/schema.js:76]: ping

Comment (Bitcoinera) [reply api/lib/schema.js:76]: Yeah, I think this is already in anyway. Was part of #4887

Comment (Bitcoinera) [reply api/lib/schema.js:76]: yes, it is. closing this one now

**Pull request closed without merging**